examples/jupyter/candu.ipynb

```diff
@@ -399,688 +399,9 @@
-       "22       3   44       1  403       2    5          22   total  flux 1.11e-01   \n",
-       "23       3   44       1  404       2    5          23   total  flux 1.12e-01   \n",
-       "24       3   44       1  405       2    5          24   total  flux 1.10e-01   \n",
-       "25       3   44       1  406       2    5          25   total  flux 1.00e-01   \n",
-       "26       3   44       1  407       2    5          26   total  flux 9.54e-02   \n",
-       "27       3   44       1  408       2    5          27   total  flux 9.26e-02   \n",
-       "28       3   44       1  409       2    5          28   total  flux 9.55e-02   \n",
-       "29       3   44       1  410       2    5          29   total  flux 1.14e-01   \n",
-       "30       3   44       1  411       2    5          30   total  flux 1.08e-01   \n",
-       "31       3   44       1  412       2    5          31   total  flux 1.07e-01   \n",
-       "32       3   44       1  413       2    5          32   total  flux 1.12e-01   \n",
-       "33       3   44       1  414       2    5          33   total  flux 1.15e-01   \n",
-       "34       3   44       1  415       2    5          34   total  flux 1.14e-01   \n",
-       "35       3   44       1  416       2    5          35   total  flux 1.14e-01   \n",
-       "36       3   44       1  417       2    5          36   total  flux 1.11e-01   \n",
-       "\n",
-       "   std. dev.  \n",
-       "              \n",
-       "              \n",
-       "0   7.04e-03  \n",
-       "1   5.27e-03  \n",
-       "2   7.82e-03  \n",
-       "3   6.47e-03  \n",
-       "4   6.43e-03  \n",
-       "5   4.89e-03  \n",
-       "6   3.85e-03  \n",
-       "7   5.82e-03  \n",
-       "8   8.30e-03  \n",
-       "9   3.09e-03  \n",
-       "10  5.91e-03  \n",
-       "11  5.31e-03  \n",
-       "12  6.65e-03  \n",
-       "13  3.67e-03  \n",
-       "14  4.73e-03  \n",
-       "15  6.54e-03  \n",
-       "16  5.94e-03  \n",
-       "17  5.73e-03  \n",
-       "18  4.58e-03  \n",
-       "19  4.47e-03  \n",
-       "20  5.45e-03  \n",
-       "21  4.72e-03  \n",
-       "22  4.21e-03  \n",
-       "23  5.08e-03  \n",
-       "24  4.15e-03  \n",
-       "25  5.08e-03  \n",
-       "26  3.62e-03  \n",
-       "27  4.00e-03  \n",
-       "28  4.02e-03  \n",
-       "29  9.53e-03  \n",
-       "30  7.24e-03  \n",
-       "31  5.72e-03  \n",
-       "32  5.00e-03  \n",
-       "33  6.24e-03  \n",
-       "34  4.92e-03  \n",
-       "35  5.32e-03  \n",
-       "36  5.05e-03  "
-      ]
-     },
-     "execution_count": 17,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "output_tally = sp.get_tally()\n",
-    "output_tally.get_pandas_dataframe()"
+    "with openmc.StatePoint('statepoint.{}.h5'.format(settings.batches)) as sp:\n",
+    "    output_tally = sp.get_tally()\n",
+    "    output_tally.get_pandas_dataframe()"
```

Comment:
This no longer shows the dataframe. I think you'll need something like:
```
with blahblah:
    output_tally = sp.get_tally()
    df = output_tally.get_pandas_dataframe()
df
```

Reply:
Good catch! Thanks!